frontend/src/store/podcast/reducer.ts

```diff
@@ -0,0 +1,28 @@
+import { createReducer } from '@reduxjs/toolkit';
```

Comment:
your page is responsible for creating or editing a podcast. Therefore, the reducer and actions should be called `configurate-pocast`